ExtraMarkups.json

```
@@ -3,7 +3,7 @@
   "build_dependencies": [],
   "build_subdirectory": ".",
   "category": "Utilities",
-  "scm_revision": "main",
+  "scm_revision": "SlicerStable",
```

`SlicerStable` is not a suitable branch name as its meaning changes in time. Right now SlicerStable means 5.8 but in 4 months it will mean 5.10. I would recommend to follow the convention that is used in almost all extensions: use the Slicer branch name as branch name in the extension's repository.
```suggestion
  "scm_revision": "5.8",
```

---

Yep, you're right, it's fixed now.